frontend/src/scenes/experiments/Metrics/TrendsMetricForm.tsx

```diff
@@ -98,7 +98,15 @@ export function TrendsMetricForm({ isSecondary = false }: { isSecondary?: boolea
                                     showSeriesIndicator={true}
                                     entitiesLimit={1}
                                     showNumericalPropsOnly={true}
-                                    onlyPropertyMathDefinitions={[PropertyMathType.Sum]}
+                                    allowedMathTypes={[
```

Comment:
Can we define this list in one place and use across both?

Reply:
Fixed in https://github.com/PostHog/posthog/pull/27589/commits/daa9008fc2054fa6dc90ab0afa62ab4a9d751ce9